File: C++/Arrays/Problem11.cpp

// Finding Maximum consucative 1s in the array


#include <iostream>
using namespace std;

int main() 
{
    int maxCount=0;
    int n=6;
    int arr[n]={1, 0, 1, 1, 0, 1};
    int count=0;
    
    for(int i=0;i<n;i++){
      if(arr[i]==1){
        count++;
      }
      else{
        count=0;
      }
      maxCount=max(maxCount,count);
    }
    std::cout << maxCount << std::endl;
    return 0;
}

File: Javascript/Arrays/Problem11.js
// Find Maximum consicative 1 in the array





let arr=[1, 0, 1, 1, 0, 1];

let count=0;
let maxCount=0
for(let i=0;i<arr.length;i++){
  
  if(arr[i]==1){
    count++;
  }
  maxCount=Math.max(maxCount,count)
  if(arr[i]!==1){
    count=0;
  }
  
}

console.log(maxCount)
